tutorials/light_config.md

```diff
@@ -0,0 +1,92 @@
+\page light_config Light config
```

Comment:
The tutorial is also teaching how to create a model, so the title seems a bit misleading. Some ideas:

* Leave this tutorial being about lights, and move the model creation part to another tutorial (you could create a light from a string instead)
* Leave this tutorial being about creation, and move light configuration to another tutorial